lib/utils/args.js

```diff
@@ -59,7 +59,7 @@ function _normaliseFlagsAndArs(argsIntoCommand, cmd){
     if (key.match(/[0-9]/)){
       var position = parseInt(key, 10),
         arg = argsIntoCommand[position];
-      var hasArgumentAtPosition = !isNaN(position) && typeof arg !== 'undefined' && !arg.match(/^--/);
+      var hasArgumentAtPosition = !isNaN(position) && typeof arg !== 'undefined' && !arg.match(/^--/) && arg.indexOf('=') === -1;
```

Comment:
this is an extra check to make sure the value at this index is not like `-alias=key` or `--alias=key`